base_geoengine_demo/models/geo_npa.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2011-2012 Nicolas Bessi (Camptocamp SA)
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class NPA(models.Model):
+
+    """GEO OSV SAMPLE"""
+
+    _name = "dummy.zip"
+    _description = "Geoengine demo ZIP"
+
+    priority = fields.Integer(default=100)
+    name = fields.Char("ZIP", index=True, required=True)
+    city = fields.Char(index=True, required=True)
+    the_geom = fields.GeoMultiPolygon("NPA Shape")
+    # the_geom_poly = fields.GeoPolygon()
+    # the_geom_multiLine = fields.GeoMultiLine()
+    # the_geom_multipoint = fields.GeoMultiPoint()
```

Comment:
to remove? or are those here as examples?

Reply:
as examples